From b3f772574244fc6c9d3e37dd10c45ee78ea7c980 Mon Sep 17 00:00:00 2001
From: szymonPa <szymon.parapura@gmail.com>
Date: Mon, 10 Oct 2022 10:59:21 +0200
Subject: [PATCH] My first patch

---
 userprog.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/userprog.c b/userprog.c
index bebfefb..e9a8713 100644
--- a/userprog.c
+++ b/userprog.c
@@ -3,7 +3,7 @@
 
 int main() {
 
-	printf("Hello worldfff");
+	printf("Hello worldfff - modification");
 
 	return 0;
 }
-- 
2.17.1

